app/views/layouts/application.html.slim

```diff
@@ -18,7 +18,8 @@ html
       - if content_for(:content_form_errors)
         .alert.alert-warning= I18n.t("activerecord.shared.errors")
         = yield(:content_form_errors)
-            
+
     = content_for?(:content) ? yield(:content) : yield
 
     = render("footer")
+    = javascript_include_tag('analytics')
```

Comment:
this is not necessary as app/assets/javascripts does a `require_tree .` so all JS there is available everywhere

Reply:
ah, that's my misunderstanding. i had thought we wanted non-render js to load afterwards.
if all of app/assets/javascripts is being pulled in anyways then i can remove it 😄